src/protocol/message/constants.rs

```diff
@@ -11,33 +13,35 @@ pub const MAX_MESSAGE_LEN: usize = 2 * 1024 * 1024;
 /// The current network protocol version number.
 pub const PROTOCOL_VERSION: u32 = 170_100;
 /// The current network version identifier.
-pub const MAGIC_TESTNET: [u8; 4] = [0xfa, 0x1a, 0xf9, 0xbf];
-pub const MAGIC_MAINNET: [u8; 4] = [0x24, 0xe9, 0x27, 0x64];
+pub const MAGIC_TESTNET: [u8; MAGIC_LEN] = [0xfa, 0x1a, 0xf9, 0xbf];
+pub const MAGIC_MAINNET: [u8; MAGIC_LEN] = [0x24, 0xe9, 0x27, 0x64];
 
 #[cfg(test)]
-pub const MAGIC: [u8; 4] = MAGIC_TESTNET;
+pub const MAGIC: [u8; MAGIC_LEN] = MAGIC_TESTNET;
 #[cfg(all(not(test), not(feature = "crawler")))]
 pub const MAGIC: [u8; 4] = MAGIC_MAINNET;
 #[cfg(feature = "crawler")]
-pub const MAGIC: [u8; 4] = MAGIC_MAINNET;
+pub const MAGIC: [u8; MAGIC_LEN] = MAGIC_MAINNET;
+
+pub const COMM_LEN: usize = 12;
```

Comment:
probably better to use the full name, so it's more clear wherever it's user

Reply:
Done